### PR TITLE
feat: mobile search panel

### PR DESCRIPTION
**Pull request by capJavert**
### Describe what this PR does
- style adjustments for displaying mobile search panel
- also some fixes to interactions now that we have touch 
- intermediate look for app header since mobile ux work will be done later, [see](https://dailydotdev.slack.com/archives/C063RS0QZTM/p1706621190327189)
- our current header is now also fixed

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

AS-36 #done

### CODE REVIEW

File: packages/shared/src/components/icons/Clear/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React, { ReactElement } from 'react';
+import Icon, { IconProps } from '../../Icon';
+import OutlinedIcon from './outlined.svg';
+import FilledIcon from './filled.svg';
+
+export const ClearIcon = (props: IconProps): ReactElement => (
+  <Icon {...props} IconPrimary={OutlinedIcon} IconSecondary={FilledIcon} />
+);
+
+export default ClearIcon;
```
Comment (rebelchris) [inline packages/shared/src/components/icons/Clear/index.tsx:10]: Can you maybe remove the default export.
Just changed all the icons to only have named ones :)

File: packages/shared/src/components/search/SearchPanel/SearchPanelPostSuggestions.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const PanelItem = ({
     <SearchPanelItem {...rest} icon={<SearchIcon />} {...itemProps}>
       {!!purifySanitize && (
         <span
-          className="text-theme-label-tertiary typo-subhead [&>strong]:text-white"
+          className="typo-subhead [&>strong]:text-white"
```
Comment (rebelchris) [inline packages/shared/src/components/search/SearchPanel/SearchPanelPostSuggestions.tsx:38]: First time even seeing this selector 🤯

**Review by rebelchris — APPROVED** [review]
One minor request.
Nothing blocking from my side.

File: packages/shared/src/components/search/SearchPanel/SearchPanelPostSuggestions.tsx (hunk 2 above)
Comment (capJavert) [reply packages/shared/src/components/search/SearchPanel/SearchPanelPostSuggestions.tsx:38]: Right? 😆 I was surprised you can do it in tailwind I almost went to raw css file..

File: packages/shared/src/components/icons/Clear/index.tsx (hunk 1 above)
Comment (capJavert) [reply packages/shared/src/components/icons/Clear/index.tsx:10]: Yeah, I have to rebase.

**Pull request merged**